client/components/Facilitator/Components/Cohorts/Cohort.jsx

```diff
@@ -232,6 +237,21 @@ export class Cohort extends React.Component {
                                 onClick={onClick}
                             />
                         </ConfirmAuth>
+                        {currentUserInCohort && (
```

Comment:
currentUserInCohort is undefined initially, so check for it before trying to display this component.